### DISCUSSION TITLE
Show an icon to the right of a tab

### DISCUSSION THREAD

**Discussion by maggie44** [quasarframework/quasar#14260] [General - Components / Directives / etc] [answered]
Seems like there are two options to provide an icon (in this case a close icon) on a tab shown here in the codepen:

https://codepen.io/maggie0002/pen/MWVdwRe

Can anyone think of a way to show it directly to the right of the text so they all sit on one row?

**Comment by MilosPaunovic** [comment, marked as answer]
https://codepen.io/milospaunovic/pen/qBoGOqK?editors=1010

**Reply by maggie44** [reply]
Urgh, sometimes after someone has mentioned it, it just seems so obvious! Exactly what I need, thanks.
